scout/server/blueprints/dashboard/controllers.py

```diff
@@ -189,90 +151,101 @@ def get_general_case_info(adapter, institute_id=None, slice_query=None):
         "individuals": 1,
         "tags": 1,
     }
+
     cases = adapter.cases(
-        owner=institute_id, name_query=name_query, projection=CASE_GENERAL_INFO_PROJECTION
+        owner=institute_id, name_query=cases_form, projection=CASE_GENERAL_INFO_PROJECTION
     )
 
-    case_counter_keys = [
-        "phenotype",
-        "causative",
-        "pinned",
-        "cohort",
-        "tagged",
-    ]
-    case_counter_keys.extend(CASE_TAGS.keys())
+    # Initialize counters and structures
+    case_counter_keys = ["phenotype", "causative", "pinned", "cohort", "tagged"] + list(
+        CASE_TAGS.keys()
+    )
+    case_counter = initialize_case_counter(case_counter_keys)
+    pedigree = initialize_pedigree()
```

Comment:
This could be a constructor for a nice data object. But the abstraction is the same, so all good.